provers/celestia-prover/README.md

```diff
@@ -4,15 +4,15 @@ The Celestia Prover is a gRPC service that generates zero-knowledge proofs for C
 
 ## Usage
 
-> [!WARNING]
+> ⚠️ **Warning**
 > This gRPC service is still under development and may not work as described
 
 To run the server you will need to clone the repo and install rust and cargo. To run the node you also need to set the following environment variables:
 
 - `TENDERMINT_RPC_URL` - the url of the tendermint chain you are proving.
 - `RPC_URL` the json rpc url of the evm chain you are generating the proofs for.
 
-To then run the server (on port `:50051`):
+To then run the server (on port `:50001`):
```

Comment:
The celestia-prover starts on port 50051 so this change seems like a mistake 
```suggestion
To then run the server (on port `:50051`):
```

Reply:
previously, it was 50001. there has been some changes since i started working on this pr.

Reply:
It's been 50051 since the first celestia-prover PR in this repo: https://github.com/celestiaorg/celestia-zkevm-ibc-demo/pull/33/files#diff-3e3eeb59d344e85d64268133b460af890852ed1aa94c6e049556a9dc3c2aa60cR16

I think it got accidentally modified in this PR

Reply:
hmm this PR contains commits from a while back so I think it's possible this PR didn't resolve a merge conflict correctly when pulling in `main`

Reply:
old commits are due to callum initially banrching off of an older working branch. rebasing was a total mess so i merged directly and manually resolved a bunch of conflicts